### PR TITLE
CLOUDP-134983: [AtlasCLI] Add command to allow customer to open config file on their default editor

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

_Jira ticket:_ CLOUDP-134983

## Checklist
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [x] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [x] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [x] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by blva — DISMISSED** [review]
LGTM 🚀

File: docs/atlascli/command/atlas-config.txt [hunk 1]
```diff
@@ -58,6 +58,7 @@ Related Commands
 
 * :ref:`atlas-config-delete` - Delete a profile.
 * :ref:`atlas-config-describe` - Return a specific profile.
+* :ref:`atlas-config-edit` - Opens the the config with the default text editor.
```
Comment (sarahsimpers) [inline docs/atlascli/command/atlas-config.txt:61]:
```suggestion
* :ref:`atlas-config-edit` - Opens the config file with the default text editor.
```

File: docs/mongocli/command/mongocli-config-edit.txt [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Opens the the config with the default text editor.
```
Comment (sarahsimpers) [inline docs/mongocli/command/mongocli-config-edit.txt:15]:
```suggestion
Opens the config file with the default text editor.
```

File: docs/mongocli/command/mongocli-config.txt [hunk 3]
```diff
@@ -87,6 +87,7 @@ Related Commands
 
 * :ref:`mongocli-config-delete` - Delete a profile.
 * :ref:`mongocli-config-describe` - Return a specific profile.
+* :ref:`mongocli-config-edit` - Opens the the config with the default text editor.
```
Comment (sarahsimpers) [inline docs/mongocli/command/mongocli-config.txt:90]:
```suggestion
* :ref:`mongocli-config-edit` - Opens the config file with the default text editor.
```

File: internal/cli/config/edit.go [hunk 4]
```diff
@@ -47,12 +47,11 @@ func EditBuilder() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "edit",
 		Short: "Opens the the config with the default text editor.",
-		Long:  `Will use the default editor to open the config file. You can use EDITOR or VISUAL envs to change the default.`,
+		Long:  `Uses the default editor to open the tool config file. You can use EDITOR or VISUAL envs to change the default.`,
```
Comment (sarahsimpers) [inline internal/cli/config/edit.go:50]: Question: why did we add "tool" here?

**Review by sarahsimpers — CHANGES_REQUESTED** [review]
A few copy nits, thank you!

File: internal/cli/config/edit.go (hunk 4 above)
Comment (gssbzn) [reply internal/cli/config/edit.go:50]: why not? just to make it clearer but open to suggestions, this commands was undocumented so it has never been reviewed

**Review by sarahsimpers — APPROVED** [review]

**Pull request merged**